app/models/group.rb

```diff
@@ -0,0 +1,19 @@
+# Represents a group of users.
+class Group < ApplicationRecord
+  # Associations
+  has_and_belongs_to_many :users
+  has_one :list # Only have one if the group contains viewers for a list.
```

Comment:
What's this relationship here? Are we envisioning groups as

1. A person shares their singular list with a group of people
2. Several people join a group and share their lists with one another

I was originally envisioning our data model as 2 as that would make it easier for people to mark and view what's been purchased for others. Groups are more of an iteration 2 goal so we can discuss further.

Reply:
The relationship here allows for the group to be associated with a given list, but does not require that all groups be associated to a list (i.e. if we wanted to be able to use a group for something other than just establishing viewers for a group, we may not require a corresponding list).